client/index.js

```diff
@@ -17,7 +18,6 @@ var util = require('./util')
 global.WEBTORRENT_ANNOUNCE = [ 'wss://tracker.webtorrent.io' ]
 
 var VIDEO_MEDIASOURCE_EXTS = [ '.mp4', '.m4v', '.webm' ]
-var AUDIO_MEDIASOURCE_EXTS = [ '.mp3' ]
```

Comment:
mp3 files used to stream progressively (without support for seeking). won't this change make it wait for the full file?